### PR TITLE
adding nba_boxplot function and test function

### PR DESCRIPTION
**Pull request by carlinakim**

### CODE REVIEW

File: tests/testthat/test-nba_boxplot.R [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+  test_that("Not inputting a position or teams input will result in a error, must input one", {
+    expect_error(nba_boxplot(nba_2018, stats= "GP"))
+  })
+
+  test_that("Inputting both position or teams input will result in a error, must input one", {
+    expect_error(nba_boxplot(nba_2018, position= "POS", teams= c("ORL", "UTAH", "LAC", "MIN", "BOS"), stats= "GP"))
+  })
+}
+
+
```
Comment (vanandsh) [inline tests/testthat/test-nba_boxplot.R:36]: Please call the function defined - 
```suggestion
test_nba_boxplot()
```

**Review by vanandsh — CHANGES_REQUESTED** [review]
Code coverage is not 100%. 

Missing conditions in the test function:
- position and teams are both NULL
- incorrect column name in stats
- non-numeric columns passed in stats

File: tests/testthat/test-nba_boxplot.R (hunk 1 above)
Comment (carlinakim) [reply tests/testthat/test-nba_boxplot.R:36]: Thanks for the suggestions vanand!
Have added them for 100% test coverage

Comment (vanandsh) [reply tests/testthat/test-nba_boxplot.R:36]: Thank you @carlinakim ! Since there is a new pull request, is it okay to delete this PR?

Comment (carlinakim) [reply tests/testthat/test-nba_boxplot.R:36]: Sounds good @vanandsh  , deleting this PR

**Pull request closed without merging**